l10n_pa_withholding/model/account_invoice.py

```diff
@@ -191,3 +201,90 @@ def wihholding_reconciliation(self):
             line_ids.reconcile_partial()
 
         return True
+
+
+class AccountInvoiceTaxWh(models.Model):
+    """Invoice Tax Withholding"""
+    _name = 'account.invoice.tax.wh'
+    _inherit = 'account.invoice.tax'
```

Comment:
There's no need for `_rec_name` I am prototyping from `account.invoice.tax` which already has `name`.

Or should I?
<img width="868" alt="models" src="https://cloud.githubusercontent.com/assets/7598010/19878599/1af6790e-9fc1-11e6-8ff6-6ca0a77836df.png">